### PR TITLE
Update w3c_website_templates_bundle+intl-icu.zh-hans.yaml

### PR DESCRIPTION
**Pull request by jennyliang220**
Finished remaining part of website template.

### CODE REVIEW

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 1]
```diff
@@ -16,13 +16,18 @@ page:
...
-        private: __page.acl.private
-        user: __page.acl.user
+        member: 当前页面仅 W3C 会员可见
+        team: 当前页面仅 W3C 员工可见
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">World Wide Web Consortium</a>.<br>
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:25]:
```suggestion
        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 2]
```diff
@@ -16,13 +16,18 @@ page:
...
+          <abbr title="World Wide Web Consortium">W3C</abbr><sup>&reg;</sup>
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:26]:
```suggestion
          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> 
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 3]
```diff
@@ -16,13 +16,18 @@ page:
...
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>,
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:27]:
```suggestion
          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 4]
```diff
@@ -16,13 +16,18 @@ page:
...
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:28]:
```suggestion
          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 5]
```diff
@@ -16,13 +16,18 @@ page:
...
+          <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document" 
+          title="W3C Software and Document Notice and License">permissive document license</a> rules apply.
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:30]:
```suggestion
          title="W3C软件和文档声明和许可政策">文档许可政策</a>。
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 6]
```diff
@@ -16,13 +16,18 @@ page:
...
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> 
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及
+          <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document" 
+          title="W3C软件和文档声明和许可政策">文档许可政策</a>。
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:30]:
```suggestion
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 7]
```diff
@@ -16,13 +16,18 @@ page:
...
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> 
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及
+          <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:29]:
```suggestion
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 8]
```diff
@@ -16,13 +16,18 @@ page:
...
+        team: 当前页面仅 W3C 员工可见
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> 
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:28]:
```suggestion
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 9]
```diff
@@ -16,13 +16,18 @@ page:
...
+        member: 当前页面仅 W3C 会员可见
+        team: 当前页面仅 W3C 员工可见
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> 
+          <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:27]:
```suggestion
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 10]
```diff
@@ -16,13 +16,18 @@ page:
...
-        user: __page.acl.user
+        member: 当前页面仅 W3C 会员可见
+        team: 当前页面仅 W3C 员工可见
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+          遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup>
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:26]:
```suggestion
        遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、<a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及<a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document" title="W3C软件和文档声明和许可政策">文档许可政策</a>。
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 11]
```diff
@@ -16,13 +16,14 @@ page:
...
-        user: __page.acl.user
+        member: 当前页面仅 W3C 会员可见
+        team: 当前页面仅 W3C 员工可见
+        private: 当前页面可见范围有限
+        user: 当前页面仅本人可见
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        版权所有 &copy; {date, date, ::y} <a href="https://www.w3.org/">万维网联盟</a>。<br>
+        遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、<a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及<a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document" title="W3C软件和文档声明和许可政策">文档许可政策</a>。
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:26]:
```suggestion
        遵循 <abbr title="万维网联盟">W3C</abbr><sup>&reg;</sup><a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">责任声明</a>、<a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">标志使用</a>以及<a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document" title="W3C软件和文档声明和许可政策">文档许可政策</a>。
```

**Comment by jennyliang220** [comment]
Result preview:
image

**Review by jean-gui — APPROVED** [review]

**Review by xueyuanjia — APPROVED** [review]

**Pull request merged**